google-home-page/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Google</title>
+    <link rel="icon" href="google-favicon.png" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+  </head>
+  <body>
+    <section class="top">
+      <span
+        ><a href="https://mail.google.com/">Gmail</a>
+        <a href="https://www.google.com/imghp?hl=en&authuser=0&ogbl">Images</a>
+        <img id="apps" src="google-apps.png" alt="google apps" />
+        <img id="guest" src="guest-user.png" alt="your gmail account" />
+      </span>
+    </section>
+    <!--Main segment Starts-->
+    <div class="center">
+      <img class="middle" id="google" src="main-logo.png" alt="google logo" />
+      <p id="borders"></p>
```

Comment:
A `p` tag means there's a text that's going to come. You can use `span` tag here alternatively